src/System.Drawing.Common/src/System/Drawing/Graphics.Unix.cs

```diff
@@ -1046,7 +1039,7 @@ public void DrawPath(Pen pen, GraphicsPath path)
                 throw new ArgumentNullException("pen");
             if (path == null)
                 throw new ArgumentNullException("path");
```

Comment:
nameof? unless it is slotted for the next PR.

Reply:
this may help - https://github.com/dotnet/corefx/pull/30228#issuecomment-395936113

Reply:
I wasn't updating the Unix files in this PR as some of them will fold into shared code.